camera_topic_name: /camera/color/image_raw
# model_path: $(find camera_apps)/model
conf_threshold: 0.4
# conf_threshold: 0.3
# hit_threshold: 1.2
# hit_threshold: 1.0
hit_threshold: 0.9
